docs/docs/styled-components.md

```diff
@@ -112,7 +112,29 @@ export default () => (
 )
 ```
 
-### Enabling user stylesheets with a stable class name
+## Creating Global Styles
+
+Styled-components are primarily used for a single CSS class that is isolated from other components. In some cases, you want to override global styling — for example, the default margins of your `body` element. Styled-components has your back. You can use the `createGlobalStyle` to accomplish this. We advise using `createGlobalStyle` in Layout components, which are shared over multiple pages rather than using it on a single page.
+
+The example below shows how to create a `GlobalStyle` (which is a StyledComponent) for the color purple by importing `createGlobalStyle` from `styled-components`.
+
+```jsx:title=src/pages/index.js
+import { createGlobalStyle } from 'styled-components'
```

Comment:
```suggestion
import React from "react"
import { createGlobalStyle } from 'styled-components'
```
Adding React here so that people don't get tripped up by it!